v4/export/config.go

```diff
@@ -62,6 +71,9 @@ type Config struct {
 	SessionParams      map[string]interface{}
 
 	PosAfterConnect bool
+
+	externalStorage     storage.ExternalStorage
```

Comment:
We need to append a `json:"-"` tag to the end of this field so that it won't be printed, just like we did for field `TableFilter`.